modules/rosa-configure.adoc

```diff
@@ -13,6 +13,11 @@ Use the following commands to configure the {product-title} (ROSA) CLI, `rosa`.
 
 Log in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token from link:https://console.redhat.com/openshift/token/rosa[the ROSA token page].
 
+[NOTE]
```

Comment:
I take back my previous comment. Take a closer look, I understand this is different. 

Checking with engineering to see if this is intended feature. We will need confirmation from QA as well.